pages/gsoc/index.md

```diff
@@ -2,6 +2,7 @@
 layout: col-sidebar
 title: GSoC
 tags: gsoc
+permalink:gsoc
```

Comment:
Looks like this front matter is failing to parse correctly because I left out the space on permalink, will fix this on the next PR.